packages/@coorpacademy-components/src/template/back-office/brand-update/style.css

```diff
@@ -11,21 +12,32 @@
 @value wide from breakpoints;
 
 .container {
-  width: 100%;
   height: 100%;
   scroll-behavior: unset;
   margin: 0 auto;
-  padding-bottom: 80px 20px;
   display: flex;
-  font-family: "Open Sans";
+  flex-direction: row;
 }
 
-.header {
-  margin: 24px 0 56px 0;
+.left {
+  width: 223px;
 }
 
-.headerForNotifications {
-  margin: 24px 0 0 0;
+.contentWrapper {
+  width: calc(100% - 351px);
```

Comment:
Not sure that `calc` is well supported by IE11 :(

Reply:
@Fausto95 ce qui n'est pas supporté par IE c'est les calc nestés. C'est pas le cas ici.
<img width="470" alt="Capture d’écran 2021-11-29 à 16 18 24" src="https://user-images.githubusercontent.com/7602475/143894133-26ccc189-0830-4e1d-8441-d9c09ff0197a.png">